QualityControl/public/pages/objectView/ObjectViewModel.js

```diff
@@ -127,6 +132,38 @@ export default class ObjectViewModel extends BaseViewModel {
     return `/api/object/proxy/download/?token=${this.model.session.token}&objectIds=${objectId}`;
   }
 
+  /**
+   * Get the current display state of object information.
+   * @returns {boolean} - `true` if object information is currently displayed, `false` otherwise.
+   * @example
+   * const isVisible = objectViewModel.getObjectInfoVisible();
+   */
+  getObjectInfoVisible() {
+    return this.objectInfoVisible;
+  }
+
+  /**
+   * Set the display state of object information.
+   * Notifies any observers/listeners about the change.
+   * @param {boolean} objectInfoVisible - `true` to show object information, `false` to hide.
+   * @example
+   * objectViewModel.setObjectInfoVisible(true);
+   */
+  setObjectInfoVisible(objectInfoVisible) {
+    this.objectInfoVisible = objectInfoVisible;
+    this.notify();
+  }
+
+  /**
+   * Toggle the display state of object information.
+   * If currently visible, it becomes hidden; if hidden, it becomes visible.
+   * @example
+   * objectViewModel.toggleObjectInfoVisible();
+   */
+  toggleObjectInfoVisible() {
+    this.setObjectInfoVisible(!this.getObjectInfoVisible());
```

Comment:
Within the class where the variable is defined as private, we should make use of the variable itself rather than the getter